examples/quickstart-pytorch/README.md

```diff
@@ -72,3 +74,29 @@ python3 client.py --partition-id 1
 ```
 
 You will see that PyTorch is starting a federated training. Look at the [code](https://github.com/adap/flower/tree/main/examples/quickstart-pytorch) for a detailed explanation.
+
+______________________________________________________________________
```

Comment:
remove? (also those in https://github.com/adap/flower/pull/3117?)

Reply:
Sure. I don't have a strong opinion about this